### ISSUE TITLE
static authenticator embedded in README

### ISSUE THREAD

**Issue by doomcrata** [socrata/arcs#12]
RE https://socrata.atlassian.net/browse/OPS-2340

./README.md:export CROWDFLOWER_API_KEY=LbcxvIlE3x1M8F6TT5hN

**Comment by rlvoyer** [comment]
@kavedder can confirm, but I believe that's a fake key.

**Comment by jasonij** [comment]
If it's a fake key, it should be obviously fake, like =THISisAf4k3K3Y or something.

**Comment by kavedder** [comment]
Yeah, it's just the one that the Python crowdflower implementation used in their docs. https://pypi.python.org/pypi/crowdflower
